paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -91,41 +102,28 @@ class FormViewModel(
             val element = when (type) {
                 Field.NameInput -> TextFieldElement(NameConfig()) // All configs should have the label passed in for consistency
                 Field.EmailInput -> TextFieldElement(EmailConfig())
+                Field.CountryInput -> DropdownElement(CountryConfig())
             }
             listCompleteFlows.add(element.isComplete)
-            listOfPairs.add(element.input.map { Pair(type.paymentMethodCreateParamsKey, it) })
+            listOfPairs.add(element.paymentMethodParams.map {
+                Pair(
+                    type.paymentMethodCreateParamsKey,
+                    it
+                )
```

Comment:
Can you use a `data class` instead of `Pair`? I find `Pair` makes code harder to understand.

Reply:
This will be a pair of field and the value

Reply:
With pair there are some nice functions we can use.